src/components/services/AdGuardHome.vue

```diff
@@ -16,14 +16,17 @@
             </div>
             <div class="media-content">
               <p class="title is-4">{{ item.name }}</p>
-              <p class="subtitle is-6">{{ item.subtitle }}</p>
+              <p class="subtitle is-6">
+                <template v-if="item.subtitle">
+                  {{ item.subtitle }}
+                </template>
+                <template v-else-if="stats">
+                  {{ percentage }}&percnt; blocked
+                </template>
+              </p>
             </div>
-            <div
-              v-if="status"
-              class="status"
-              v-bind:class="status.protection_enabled ? 'enabled' : 'disabled'"
-            >
-              {{ status.protection_enabled }}
+            <div v-if="!item.subtitle" class="status" :class="protection">
```

Comment:
Why the `v-if` here ? the status can be displayed with or without a subtitle.

Reply:
You are absolutely right, the status is always being fetched anyway unlike the stats. I have just commited the code change accordingly.